search_log_test.go

```diff
@@ -515,3 +534,102 @@ func (s *searchLogSuite) TestParseLogItem(c *C) {
 		c.Assert(item.Message, Equals, cas.message)
 	}
 }
+
+func (s *searchLogSuite) TestReadAndAppendLogFile(c *C) {
```

Comment:
This test output is not expected. I think we need to re-implement the `readLineReverse` function.

Reply:
Yep, I decided to continue to re-implement it in this PR.

Reply:
Done!